<?xml version="1.0"?>

<launch>
      <arg name="map_name" default="ramona"/>
      <arg name="map_resolution" default="0.05"/>
      <arg name="inflation_radius" default="0.1"/>

      <node pkg="planning3d" type="grid_map_3D_publisher.py" name="grid_map_3D_publisher" output="screen">
      <param name="map_file_path" value="$(find maps)/$(arg map_name).world.json"/> 
      <param name="map_resolution" value="$(arg map_resolution)"/> 
      <param name="inflation_radius" value="$(arg inflation_radius)"/> 
      </node>

      <node pkg="planning3d" type="path_server_3D.py" name="path_server_3D" output="screen">
      <param name="map_file_path" value="$(find maps)/$(arg map_name).world.json"/> 
      <param name="map_resolution" value="$(arg map_resolution)"/> 
      <param name="inflation_radius" value="$(arg inflation_radius)"/> 
      </node>
</launch>